apis/v1alpha1/opentelemetrycollector_types.go

```diff
@@ -33,10 +33,22 @@ type OpenTelemetryCollectorSpec struct {
 	// +optional
 	Args map[string]string `json:"args,omitempty"`
 
+	// Autoscale turns on/off the autoscale feature. By default, it's enabled if the Replicas field is not set.
+	// +optional
+	Autoscale *bool `json:"autoscale,omitempty"`
```

Comment:
Should this be set to true in the defaulting webhook if `replicas` is nil?

Reply:
not sure, right now the default is one replica, if you don't specify replicas, I tried to not change default behaviour